docs/src/1_Mechanics/04_Powers.yaml

```diff
@@ -918,4 +918,202 @@ Amplify Ally:
+    Description: You move silently in the shadows, and are especially deadly when unseen. 
+    XP: 2
+    Prereq:
+        Role: Martial
+        Level: 2
+        Skill: Stealth > 0
+
+Vanishing Step: 
+    Type: Major
+    Category: 
+        - Movement
+        - Stealth
+        - Assassin
+    Mechanic: Choose a point you can see within 10 spaces of you. If the space is unoccupied, you appear there Hidden. If you attempt to occupy a space with another creature in it you are not Hidden
+    Description: Sometimes, moving in the shadows is easiest when no one knows where you're going.
+    XP: 2
+    PP: 2
+    Prereq:
+        Role: Martial
+        Level: 3
+        Skill: Stealth > 1
+        Power: Hidden Strike
+    Range: 10
+    Target: 1
+    Save:
+        Trigger: If the space is occupied, the creature occupying that space makes a 
+        Type: STR
+        Fail: they are pushed out of the way
+        Succeed: instead, choose a space within 1 space of the creature to appear. 
+
+Summon Creature:
+    Type: Major
+    Category:
+        - Companion
+        - Summoner
+    Mechanic: Summon a creature to assist you. This creature uses the 'Summoned Creature' stat block and is treated as a Companion for the duration. You choose the creature's form. 
+    Description: It's good to have a friend, even when you have to summon them from somewhere else. Who ever said no to a little extra help?
+    XP: 2
+    PP: 2
+    Prereq:
+        Role: Caster
+        Level: 2
+        Skill: Knowledge > 0
+    Duration: 1 hour
+
+Phase Swap:
+    Type: Minor
+    Category:
+        - Companion
+        - Movement
+        - Summoner
+    Mechanic: Use half your speed to swap spaces with a Summoned Creature 
+    Description: It can often be better to change places with a friendly creature. You'd prefer to be there, or they'd prefer to be here.
+    XP: 2
+    PP: 1
+    Prereq:
+        Role: Caster
+        Level: 3
+        Skill: Knowledge > 1
+        Power: Summon Creature
+    Range: 10
+
+Life Link: 
+    Type: Passive
+    Category: 
+        - Companion
+        - Summoner   
+    Mechanic: While you have at least one summoned creature active, when you take damage you can choose to apply that damage to your summoned creature instead.
+    Description: Channeling energy between friends can sometimes be mutually beneficial. Other times, it's simply beneficial.
+    XP: 2
+    Prereq:
+        Role: Caster
+        Level: 3
+        Skill: Knowledge > 1
+        Power: Summon Creature
+
+Create Potion:
+    Type: Major
+    Category:
+        - Recovery
+        - Alchemist
+    Mechanic: Create a Potion of Healing. After successfully completing a Quick Rest, you can exchange one Rest Card for a Potion of Healing without expending Power Points. All Potions created last until the end of your next Full Rest.
```

Comment:
Is there a limit on the number of rests i could take? Could I use a whole day to make a number of rests equal to my rest card max, not full rest, and then pass them out to my party?

Reply:
I believe we had previously written somewhere that we were capping Quick Rests to 2x per day, but not sure if that got officially written into the rules when we resolved the other aspect of Quick Rests recently.

Reply:
Added this in a commit to specify that a player can only gain the benefits of 2 Quick Rests per Full Rest.